display/mbaDisplayBrainSlice.m

```diff
@@ -67,9 +67,11 @@
     h = surf([x.min x.max],[y.min y.max],repmat(z.min, [2 2]),...
         image_rgb,'facecolor','texture','FaceAlpha',alpha);
 end
+axis off
 
 % Rescale the axis to fit the img
 if rescale == 1
+    keyboard
```

Comment:
I think this should be removed?

Reply:
@DanNBullock line 74 seems like a bug to me also.

Reply:
If we are being completely honest, I did not make the vast majority of these changes.  I don't actually know the provenance of them.  I'm fine with whatever changes need to be made (i.e. getting rid of this debug stuff)

Reply:
@DanNBullock this is the PR stuck